partner_kpi_data/__manifest__.py

```diff
@@ -15,9 +16,13 @@
         'partner_view_360'
     ],
     'data': [
-			'views/res_partner_view.xml',
-            'security/ir.model.access.csv'
+	'views/res_partner_view.xml',
```

Comment:
Wrong indented, it works but look ugly, indentation shall be multiples of 4 spaces (add 2? spaces)